### PR TITLE
removed linting warnings

### PR DESCRIPTION
**Pull request by rijusougata13**
## Description

removed linting warnings

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by piyumaldk** [comment]
Good. I was about to send a PR related to this issue but now I can merge this after the approve.

**Review by thisuraseniya — APPROVED** [review]

**Review by Niweera — APPROVED** [review]

**Pull request merged**
